package.json

```diff
@@ -33,7 +33,6 @@
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-cucumber": "^1.4.0",
     "eslint-plugin-import": "^2.22.1",
-    "http-status-codes": "^2.1.4",
```

Comment:
Since we were using the status code and not it's meaning, we could give the code expected directly from the feature, so using the  http-status-code was redundant